payments/models.py

```diff
@@ -10,7 +10,12 @@
 from payments.enums import AdditionalProductType, PeriodType, PriceUnits, ServiceType
 from utils.models import TimeStampedModel, UUIDModel
 
-TAX_PERCENTAGES = [Decimal(x) for x in ("24.00", "10.00")]
+PLACE_PRODUCT_TAX_PERCENTAGES = [Decimal(x) for x in ("24.00",)]
+ADDITIONAL_PRODUCT_TAX_PERCENTAGES = [Decimal(x) for x in ("24.00", "10.00")]
+
+TAX_PERCENTAGES = list(
+    set(PLACE_PRODUCT_TAX_PERCENTAGES + ADDITIONAL_PRODUCT_TAX_PERCENTAGES)
+)
```

Comment:
so, we aren't really using this anywhere, right? I'd say, let's sweep it under the rug for now

Reply:
We use it [here](https://github.com/City-of-Helsinki/berth-reservations/blob/ven-396/pricing-api/payments/models.py#L36):
```python
# payments/models.py#L36
choices=[(tax, str(tax)) for tax in TAX_PERCENTAGES],
```
only for the abstract base product. We could also do the 
```python
list(
    set(PLACE_PRODUCT_TAX_PERCENTAGES + ADDITIONAL_PRODUCT_TAX_PERCENTAGES)
)
```
directly on the model choices.

Reply:
if we override the `tax_percentage` field on both child models of the abstract model, then we probably do not really need it there 🤔

Reply:
Yeah, I was thinking that... but I decided to keep the field to indicate that any product needs to have a taxable value and to have a set of defaults in case in the future, another model would not override it.

However, we can remove it if it's doing no good to anyone. :)

Reply:
I feel like it adds more complexity for people who would be new to the codebase. They would see this field and its choices and will have to track how each product overrides it and will have to understand that the field on the base class is an example of sorts or a default for potential future child classes. I think it would make people more cautious if we remove the field, so that they would see that `tax_percentage` is missing unless they add it explicitly where they need it.

Reply:
Makes sense, it would require more caution when implementing other child products.

The dream would be to have kinda "configurable" choices so we could have something like this:
```python
class AbstractTaxableProduct:
    tax_percentage = DecimalField(
        choices=self._meta.TAX_OPTIONS
    )

class ConcreteProduct(AbstractTaxableProduct):
    class Meta:
        TAX_OPTIONS = [10, 20]
```

But I guess removing it for the parent class will do for now. 😅

Reply:
configurable choices would be super cool 😻 let's see what will happen with the webshop microservice and then add more sugar like this, if we will still have this code in our codebase